### ISSUE TITLE
Code adaptations for tuning model

### ISSUE THREAD

**Issue by qinghezeng** [mahmoodlab/CLAM#18]
Hi, first thank you again for sharing these fantastic codes! I am tuning the model on my dataset to get better performance and have a few questions:

1. If we train clam with a different B value (e.g., 64) via `--B`, should we pass B to initial the model while validation? 
3. If we have multiple classes, should we modify the hard-coded class number of `instance_loss_fn = SmoothTop1SVM(n_classes = 2)` in `core_utils.py`?
4. If we enable `--no_inst_cluster`, what adaptation should we make in `core_utils.py` and `eval.py`?

**Comment by fedshyvana** [comment]
Hi, since the additional task introduced by clam is not actually used for inference (unless you are interested in the performance of the auxiliary task, which currently we only monitor during training/validation), if you train with different hyperparameters, or different loss functions, it does not affect how the model is evaluated. so for 1 and 3. I believe you don't have to make any changes (but feel free to email/msg me if you encounter problems). 
For 2. the instance_loss_fn is computed as one vs. rest for each class, so n_classes = 2 should work for any number of classes  you use for your problem.

**Comment by qinghezeng** [comment]
Thank you very much for your prompt response!
